src/main/java/team/reborn/energy/api/EnergyStorage.java

```diff
@@ -31,7 +32,7 @@
  *
  * @see Transaction
  */
-@SuppressWarnings({"unused", "deprecation", "UnstableApiUsage"})
```

Comment:
Did we end up stabilizing the transfer API? I forgot. 😄

Reply:
Yes :)